### ISSUE TITLE
[Suggestion] Extend mode for Mercurial Eye

### ISSUE THREAD

**Issue by Nova001Sol** [sinkillerj/ProjectE#650]
The Mercurial Eye, my alchemical must-have construction thing. I can build and transmute with it. The one thing I need, however, is a way to easily extend my walls and roof (oh, the ROOF!). Like ExUtils' Builder Wand, but Klein Star-powered. A Builder Wand is a fairly simple tool (as you probably know, but suggestions need detail) that allows one to quickly place blocks from their inventory. When right-clicked with on any side of any given block that the user has in their inventory, the Wand places blocks on every connected block of that same type on the side that was right-clicked on. It does only one layer at a time, and has a limited range. Connected blocks are adjacent blocks or diagonally connected blocks on the same plane that was originally right-clicked.

**Comment by sinkillerj** [comment]
Maybe something to think about for the addons mod, as of right now however the Eye does have wall mode etc.

**Comment by mFaunda** [comment]
Sinkillerj I almost explained to you how it used to work before I saw "owner" on your tag.  There's no extension mode in the version I just downloaded from CurseForge (ProjectE-1.12-PE1.3.0.jar - Apr 30, 2018).  Only normal mode and transmutation mode.  I want to make a floor bottoming off the END and I'm running into the same problem Nova is with their roof making.  Your "wall" (normal) mode still requires a block to be behind where you want the wall to actually go.  If you point me to the file path that tells the mercurial eye what it's doing I'd happily take a swing at it.
